simplekv/transform/transformer.py

```diff
@@ -0,0 +1,96 @@
+# coding=utf8
+
+
+class Transformer(object):
```

Comment:
So you propose to replace
```
transformed = transformer.transform(data) + transformer.finalize()
```
with something like
```
transformed = transformer.transform_all(data)
```
I really don't think that this reduces the complexity overall, as `Transformer` now has three methods instead of two ...